istioctl/cmd/proxyconfig.go

```diff
@@ -314,7 +314,33 @@ func proxyConfig() *cobra.Command {
 		},
 	}
 
-	configCmd.AddCommand(clusterConfigCmd, listenerConfigCmd, routeConfigCmd, bootstrapConfigCmd, endpointConfigCmd)
+	secretConfigCmd := &cobra.Command{
+		Use:   "secret <pod-name[.namespace]>",
+		Short: "Retrieves secret configuration for the Envoy in the specified pod",
+		Long:  `Retrieve information about secret configuration for the Envoy instance in the specified pod.`,
```

Comment:
Add a warning to tell users this command is experimental, see other experimental commands for the all caps warnings.